### PR TITLE
update spec to reflect recent changes to theme location

### PR DESCRIPTION
**Pull request by ryanlerch**
This commit updates the specfile to reflect the new
locations of the theme files -- static and templates

related: PR #227

### CODE REVIEW

File: fas.spec [hunk 1]
```diff
@@ -100,18 +104,29 @@ chmod 755 %{buildroot}%{python_sitelib}/%{name}/
 #%files -f %{name}.lang
 %files
 %doc README.rst COPYING fas.spec fas.wsgi
-%dir %{_datadir}/%{name}
+%{_datadir}/%{name}/theme/default/*
```
Comment (laxathom) [inline fas.spec]: You need to keep `%dir` macro to keep package's ownership on `%{_datadir}/%{name}`.

File: fas.spec [hunk 2]
```diff
@@ -100,18 +104,29 @@ chmod 755 %{buildroot}%{python_sitelib}/%{name}/
...
+%exclude /%{python_sitelib}/%{name}/theme/fedoraproject/
+%exclude /%{datadir}/%{name}/theme/fedoraproject/
+
 
 %files theme-fedoraproject
 %doc COPYING
-%{_datadir}/%{name}/themes/fedoraproject
+%exclude %{python_sitelib}/%{name}/theme/fedoraproject/static
+%{_datadir}/%{name}/theme/fedoraproject
+%exclude %{python_sitelib}/%{name}/theme/fedoraproject/
```
Comment (laxathom) [inline fas.spec]: I don't think you need to exclude those files in sub-packages as well.

**Comment by ryanlerch** [comment]
updated with the recommended changes

**Comment by laxathom** [comment]
I almost forgot, the current release actually is 3.0.1. see `fas/release.py`.
Could you update the spec while you at it?
Other than that :+1:

**Comment by ryanlerch** [comment]
updated the spec to match the release.py.

merging now.

**Pull request merged**